src/ontology/mondo-edit.obo

```diff
@@ -570254,6 +570257,14 @@ relationship: has_characteristic MONDO:0021152 ! inherited
 property_value: http://purl.org/dc/terms/creator "https://orcid.org/0000-0002-3458-4839" xsd:string
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6897" xsd:anyURI
 
+[Term]
+id: MONDO:1030002
+name: dysplasia of the proximal femoral epiphyses
+def: "A developmental disorder affecting the growth and development of the proximal end of the femur (thigh bone) near the hip joint characterized by avascular necrosis of the femoral head, cystic changes of the femoral head, and/or sclerosis of the femoral head. It is a relatively milder form of the other skeletal disorders associated with COL2A1." [https://orcid.org/my-orcid?orcid=0009-0003-2127-3550]
```

Comment:
This orcid is not valid. The requester probably copied/pasted incorrectly. 
In addition to updating this orcid, you can add the ClinGen working group URL

Reply:
Thank you, fixed now.